code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -177,7 +177,14 @@
 						makeNewConstruct(/mob/living/simple_animal/hostile/construct/wraith, A, user, 0, T.loc)
 
 					if("Artificer")
-						makeNewConstruct(/mob/living/simple_animal/hostile/construct/builder, A, user, 0, T.loc)
+						if(iscultist(user))
+							makeNewConstruct(/mob/living/simple_animal/hostile/construct/builder, A, user, 0, T.loc)
+
+						if(iswizard(user))
+							makeNewConstruct(/mob/living/simple_animal/hostile/construct/builder, A, user, 0, T.loc)
```

Comment:
the new parlance for these lines will be 
`is_antag(user, list("cultist","wizard"))` 
or 
`is_antag(user, "cultist") || is_antag(user, "wizard")`

But unless this doesn't get merged until after datum antags part 1 you can't actually use that. I'll deal with the conflicts on my end when they occur.